StreemNow/ViewController.swift

```diff
@@ -165,6 +183,39 @@ class ViewController: UIViewControllerSupport {
         }
         
     }
+    
+    func createInvite() {
+        guard let currentUser = StreemInitializer.shared.currentUser else {
+            presentAlert(message: "Error creating invite")
+            return
+        }
+        
+        var name = "StreemNow Invitee"
+        let currentUserName = currentUser.name
+        let initials = currentUserName.split(separator: " ")
+            .compactMap { $0.prefix(1).uppercased() }
+            .reduce("") { "\($0)\($1)" }
+        name += " (\(initials))"
+
+        showActivityIndicator(true)
+
+        Streem.sharedInstance.createInvitation(forUser: name, referenceId: nil, type: .link) { [weak self] invitation, error in
+            DispatchQueue.main.async { [weak self] in
+                guard let self = self else { return }
+                self.showActivityIndicator(false)
+                
+                if let invitation = invitation {
+                    DispatchQueue.main.async { [weak self] in
+                        guard let self = self else { return }
+                        self.invitationTextField.text = invitation.code.formattedAsInvitationCode()
+                        self.showActivityIndicator(false)
```

Comment:
Given that you've already called this just above, I guess you don't need to also call it here.